packages/volto/src/components/manage/Widgets/ObjectListWidget/ObjectListWidget.jsx

```diff
@@ -76,6 +76,7 @@ const ObjectListWidget = (props) => {
     onChange,
```

Comment:
We cannot move files in the structure without being a breaking change. If we want this to go to 18, we should undo this.